tests/test_mesolve.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import qutip as qt
+
+import torchqdynamics as tq
+
+
+def test_mesolve_batching():
+    """Test the batching of H and rho0 in mesolve, and the returned object sizes."""
+    # parameters
+    n = 8
+    kappa = 1.0
+    delta = 2 * np.pi
+    alpha0 = 1.0
+
+    # operators
+    a = qt.destroy(n)
+    adag = a.dag()
+    H = delta * adag * a
+    H_batched = [0.5 * delta * adag * a, delta * adag * a, 2 * delta * adag * a]
+    b_H = len(H_batched)
+    jump_ops = [np.sqrt(kappa) * a, np.sqrt(kappa) * a]
+    exp_ops = [(a + adag) / np.sqrt(2), (a - adag) / (np.sqrt(2) * 1j)]
+    num_exp_ops = len(exp_ops)
+
+    # other arguments
+    rho0 = qt.coherent(n, alpha0)
+    rho0_batched = [
+        qt.coherent(n, alpha0),
+        qt.coherent(n, 1j * alpha0),
+        qt.coherent(n, -alpha0),
+        qt.coherent(n, -1j * alpha0)
+    ]
```

Comment:
It would probably be better to not use qutip objects in this test, and to write another test that checks if qutip is supported by `mesolve`. But since we're lacking proper object definitions, this is probably fine for now.

Reply:
I think the same 😊

Reply:
It will just be a matter of replacing every `qt` by a `tq` 😋